packages/terra-slide-panel/src/SlidePanel.scss

```diff
@@ -1,121 +1,124 @@
-@import '~terra-mixins';
 @import './variables';
-@import './mixins';
 
-.terra-SlidePanel {
-  overflow: hidden;
-  position: relative;
-}
-
-.terra-SlidePanel-main,
-.terra-SlidePanel-panel {
-  background-color: #fff;
-  box-sizing: border-box;
-
-  height: 100%;
-
-  -webkit-overflow-scrolling: touch;
-  -ms-overflow-style: -ms-autohiding-scrollbar;
-  overflow-y: auto;
+/* stylelint-disable selector-class-pattern */
+:local {
+  .slide-panel {
+    overflow: hidden;
+    position: relative;
+  }
 
-  transition: transform 0.15s ease;
+  .main-panel,
```

Comment:
No problem. I changed it to .main-panel because it was initially SlidePanel-main